app/models/server/raw/LivechatDepartment.ts

```diff
@@ -5,6 +5,7 @@ import { BaseRaw } from './BaseRaw';
 import { ILivechatDepartmentRecord } from '../../../../definition/ILivechatDepartmentRecord';
 
 export class LivechatDepartmentRaw extends BaseRaw<ILivechatDepartmentRecord> {
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Can you try putting the `department` param with a `_` at the begining? That should inform TS to _ignore_ the param is not used 👀

Reply:
Alright :+1: